docs/installation/installation-using-docker-windows-10-pro-higher.md

```diff
@@ -15,9 +15,16 @@
 ## Steps
 ### 1. Create new project from Shopsys Framework sources
 ```
-composer create-project shopsys/project-base --stability=alpha --no-install --keep-vcs
+composer create-project shopsys/project-base --stability=dev --no-install --keep-vcs
 cd project-base
 ```
+**Important note:** 
+
+**Due to changes in installation process (for more see [changelog](../../CHANGELOG.md)) we recommend to use current dev-master as base for your project.**
+**Therefore there is used stability=dev in composer command above.**
+**After next release we will again move to our standard recommended stability (see notes below).**
```

Comment:
Good job on explaining the issue :+1: 
I've just fixed some grammatical mistakes and added formatting for code:

```markdown
**Due to changes in the installation process (see [changelog](/CHANGELOG.md) for details), we recommend using the current `dev-master` as a base for your project.**
**Therefore there is the `stability=dev` flag used in the `composer` command above.**
**After the next release, we will again return to our standard recommended stability (see notes below).**
```

Please, copy this to all four installation guides (docker for linux / mac / win and native) and add a note about the `TODO`'s in docs (`*.md` files) to our release checklist, so it's not forgotten.